examples/jupyter/demo010_advection.ipynb

```diff
@@ -9,9 +9,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m\u001b[1m  Activating\u001b[22m\u001b[39m project at `~/Projects/LFAToolkit.jl/examples/jupyter`\n",
+      "\u001b[32m\u001b[1mPrecompiling\u001b[22m\u001b[39m project...\n",
+      "\u001b[32m  ✓ \u001b[39mLFAToolkit\n",
+      "  1 dependency successfully precompiled in 2 seconds (126 already precompiled)\n",
+      "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
+      "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m MutableArithmetics ─ v1.0.4\n",
+      "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Polynomials ──────── v3.1.5\n",
+      "\u001b[32m\u001b[1m  No Changes\u001b[22m\u001b[39m to `~/Projects/LFAToolkit.jl/examples/jupyter/Project.toml`\n",
+      "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/Projects/LFAToolkit.jl/examples/jupyter/Manifest.toml`\n",
+      " \u001b[90m [d8a4904e] \u001b[39m\u001b[92m+ MutableArithmetics v1.0.4\u001b[39m\n",
+      " \u001b[90m [f27b6e38] \u001b[39m\u001b[92m+ Polynomials v3.1.5\u001b[39m\n"
+     ]
+    }
+   ],
```

Comment:
Need to clear the output so we aren't versioning all the churn as each person runs it.
![image](https://user-images.githubusercontent.com/3303/180558353-a776524f-2446-4e16-b388-d0017b19a029.png)